pyproject.toml

```diff
@@ -84,12 +84,34 @@ addopts = ["--import-mode=importlib"]
 
 
 [tool.mypy]
-ignore_missing_imports = true              # TODO: deactivate this
+ignore_missing_imports = true                     # TODO: deactivate this
 show_column_numbers = true
 strict = true
 exclude = [
-    "src/dsp_tools/models",                # TODO: activate this
-    "src/dsp_tools/import_scripts",        # TODO: activate this
+    "src/dsp_tools/import_scripts",               # TODO: activate this
+    "src/dsp_tools/models/exceptions.py",         # TODO: activate this
+    "src/dsp_tools/models/group.py",              # TODO: activate this
+    "src/dsp_tools/models/helpers.py ",           # TODO: activate this
+    "src/dsp_tools/models/langstring.py",         # TODO: activate this
+    "src/dsp_tools/models/listnode.py",           # TODO: activate this
+    "src/dsp_tools/models/model.py",              # TODO: activate this
+    "src/dsp_tools/models/ontology.py",           # TODO: activate this
+    "src/dsp_tools/models/permission.py",         # TODO: activate this
+    "src/dsp_tools/models/project.py",            # TODO: activate this
+    "src/dsp_tools/models/projectContext.py",     # TODO: activate this
+    "src/dsp_tools/models/propertyclass.py",      # TODO: activate this
+    "src/dsp_tools/models/propertyelement.py",    # TODO: activate this
+    "src/dsp_tools/models/resource.py",           # TODO: activate this
+    "src/dsp_tools/models/resourceclass.py",      # TODO: activate this
+    "src/dsp_tools/models/set_encoder.py",        # TODO: activate this
+    "src/dsp_tools/models/user.py    ",           # TODO: activate this
+    "src/dsp_tools/models/value.py",              # TODO: activate this
+    "src/dsp_tools/models/xmlallow.py",           # TODO: activate this
+    "src/dsp_tools/models/xmlbitstream.py",       # TODO: activate this
+    "src/dsp_tools/models/xmlpermission.py",      # TODO: activate this
+    "src/dsp_tools/models/xmlproperty.py",        # TODO: activate this
+    "src/dsp_tools/models/xmlresource.py",        # TODO: activate this
+    "src/dsp_tools/models/xmlvalue.py",           # TODO: activate this
```

Comment:
Instead of ignoring the entire "src/dsp_tools/models" folder, I want to ignore only the files that I haven't refactored yet. Like this, I make sure that the refactored ones are checked.